pkg/deploy/deploy.go

```diff
@@ -103,36 +113,24 @@ func getWordsForLogging(isDryRun bool) (action, verb string) {
 	return "Deploying", "deploy"
 }
 
-func deployConfig(configClient client.ConfigClient, apis api.APIs, entityMap *entityMap, conf *config.Config) (parameter.ResolvedEntity, []error) {
-
-	t, ok := conf.Type.(config.ClassicApiType)
+func deployConfig(configClient client.ConfigClient, apis api.APIs, entityMap *entityMap, c *config.Config, properties parameter.Properties, renderedConfig string) (parameter.ResolvedEntity, []error) {
+	t, ok := c.Type.(config.ClassicApiType)
```

Comment:
Why rename `conf` to `c`? Imo `conf` was a better var name than `c`. It's more descriptive and does not clash with any package. And I have a harder time reviewing because it's in the same commit

Reply:
One was conf, another was c. I'll bring it back to conf